README.md

```diff
@@ -33,7 +33,7 @@ mist:change-acl
 
 ## Building
 
-To build the bundle, run the following:
+To build the bundle you will need python, pip and PyYAML installed. Then just run the following:
```

Comment:
I wasn't sure the best way to install PyYAML so I just included it in the README. We need PyYAML in order to validate the config file in `scripts/validate.py`.